upup/pkg/fi/cloudup/containerd_test.go

```diff
@@ -33,20 +33,6 @@ func TestContainerdVersionUrlHash(t *testing.T) {
 		url     string
 		err     error
 	}{
-		{
-			arch:    architectures.ArchitectureAmd64,
-			version: "1.3.4",
-			url:     "https://storage.googleapis.com/cri-containerd-release/cri-containerd-1.3.4.linux-amd64.tar.gz",
-			hash:    "4616971c3ad21c24f2f2320fa1c085577a91032a068dd56a41c7c4b71a458087",
-			err:     nil,
-		},
-		{
-			arch:    architectures.ArchitectureArm64,
-			version: "1.3.4",
-			url:     "https://download.docker.com/linux/static/stable/aarch64/docker-20.10.0.tgz",
-			hash:    "6e3f80e8451ecbe7b3559247721c3e226be6b228acaadee7e13683f80c20e81c",
-			err:     nil,
-		},
```

Comment:
I am a bit puzzled about the changes in this file. Versions shouldn't matter that much. I would just leave it as it was, but would like to hear your thoughts on this.

Reply:
I've focused the testing on the latest versions containerd `1.3.x` and `1.4.x`.

I can put the code back in if there's a specific reason for testing `1.3.4`?

Reply:
These are more like unit tests that go through various functionalities of the module. Has not nothing to do with latest versions. I would prefer not to change the file at all unless there is something that changes there.

Reply:
I've reverted the deletion of the earlier versions and added `1.4.3` unit tests.